internal/uvm/scsi.go

```diff
@@ -447,6 +524,9 @@ func (sm *SCSIMount) GobDecode(data []byte) error {
 	if err := decoder.Decode(&sm.attachmentType); err != nil {
 		return fmt.Errorf(errMsgFmt, err)
 	}
+	if err := decoder.Decode(&sm.extensibleVirtualDiskType); err != nil {
```

Comment:
Do we need to rev `scsiCurrentSerialVersionID` if we are adding a new field to the serialization?

Reply:
Updated.